src/components/forms/RegisterForm.js

```diff
@@ -18,12 +18,13 @@ class RegisterForm extends Component {
     event.preventDefault();
     const { history, setAuthToken } = this.props;
     const { username, email, password } = this.state;
+    console.log(this.props);
 
     LocalApi.post("users", { username, email, password })
       .then(response => {
         setAuthToken(response.data.token);
-  //      useHistory().push("/");
-        history.push('/');
+        //      useHistory().push("/");
```

Comment:
I guess we can remove this one now.

Reply:
Sure thing. I'll remove this in my next revision! Thanks Nina :)